README.md

```diff
@@ -154,6 +154,8 @@ Configuration is optional. It should be put in a file at `config/coverage.js` (`
 
 - `excludes`: Defaults to `['*/mirage/**/*']`. An array of globs to exclude from instrumentation. Useful to exclude files from coverage statistics.
 
+- `extension`: Defaults to `['.gjs', '.gts', '.js', '.ts', '.cjs', '.mjs', '.mts', '.cts']`. Tell Istanbul to instrument only files with the provided extensions.
```

Comment:
@SergeAstapov well in the code it really is `extension` reason for this is I decided to keep it the same as the option in Isntabul since it too is singular